interfaces.go

```diff
@@ -204,13 +204,30 @@ func NewRequest(r *http.Request) *Request {
 	}
 }
 
+// Mechanism is the mechanism by which an exception was generated and handled.
+type Mechanism struct {
+	Type        string                 `json:"type,omitempty"`
+	Description string                 `json:"description,omitempty"`
+	HelpLink    string                 `json:"help_link,omitempty"`
+	Handled     *bool                  `json:"handled,omitempty"`
+	Data        map[string]interface{} `json:"data,omitempty"`
+}
+
+// SetUnhandled indicates that the exception is an unhandled exception, i.e.
+// from a panic.
+func (m *Mechanism) SetUnhandled() {
+	h := false
+	m.Handled = &h
+}
```

Comment:
I wanted to add some way to make it easier to set the `Handled` field, since it's a bit annoying otherwise. AWS SDKs, for instance, have utility functions like `func Bool(b bool) *bool`, but that felt like overkill here. Adding a corresponding `SetHandled()` also felt like it wouldn't be useful. Happy to go either (or any other) way, though.